src/LibraryViewExtension/Handlers/GetPackageResourceHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
```

Comment:
Yes, it is not required by libraryUI. I was trying this for PackageManager and it is a kind of sample implementation for web request.